app/components/ProfileCard.js

```diff
@@ -43,10 +43,71 @@ class ProfileCard extends Component {
         }).catch(err => console.error('An error occured ', err));
     }
 
+    getRankIcon(rankTier, leaderboardRank) {
+        const rankIcons = {
+            '0': require('../assets/rank_icons/rank_icon_0.png'),
+            '1': require('../assets/rank_icons/rank_icon_1.png'),
+            '2': require('../assets/rank_icons/rank_icon_2.png'),
+            '3': require('../assets/rank_icons/rank_icon_3.png'),
+            '4': require('../assets/rank_icons/rank_icon_4.png'),
+            '5': require('../assets/rank_icons/rank_icon_5.png'),
+            '6': require('../assets/rank_icons/rank_icon_6.png'),
+            '7a': require('../assets/rank_icons/rank_icon_7a.png'),
+            '7b': require('../assets/rank_icons/rank_icon_7b.png'),
+            '7c': require('../assets/rank_icons/rank_icon_7c.png'),
+        };
+
+        const starIcons = {
+            '1': require('../assets/rank_icons/rank_star_1.png'),
+            '2': require('../assets/rank_icons/rank_star_2.png'),
+            '3': require('../assets/rank_icons/rank_star_3.png'),
+            '4': require('../assets/rank_icons/rank_star_4.png'),
+            '5': require('../assets/rank_icons/rank_star_5.png'),
+        };
+
+        if (!rankTier) return null;
+        let rank, star;
+        if (rankTier > 9) {
+            if (leaderboardRank) {
+                if (leaderboardRank <= 10) {
+                    rank = '7c'; // Divine Top 10
+                } else if (leaderboardRank <= 100) {
+                    rank = '7b'; // Divine Top 100
+                }
+            }
+            if (rankTier === 76) {
```

Comment:
@dkwl93 My brother has leaderboard rank of 2372, however, his rank tier is still 75 for some reason. I'm not sure if Valve changed how leaderboard rank works. For the time being, can you add a check to Divine elite if the rankTier is 75 and leaderboardRank isn't null?

![screenshot_20180220-093426](https://user-images.githubusercontent.com/11515661/36400166-7ce4c8c6-1623-11e8-89c4-e963976e2dd9.png)

Reply:
done